src/nl/rubensten/texifyidea/structure/LatexBreadcrumbsInfo.kt

```diff
@@ -0,0 +1,28 @@
+package nl.rubensten.texifyidea.structure
+
+import com.intellij.psi.PsiElement
+import com.intellij.xml.breadcrumbs.BreadcrumbsInfoProvider
+import nl.rubensten.texifyidea.LatexLanguage
+import nl.rubensten.texifyidea.psi.LatexCommands
+import nl.rubensten.texifyidea.psi.LatexEnvironment
+import nl.rubensten.texifyidea.util.name
+
+/**
+ * @author Ruben Schellekens
+ */
+open class LatexBreadcrumbsInfo : BreadcrumbsInfoProvider() {
```

Comment:
The (deprecated) class `BreadcrumbsInfoProvider` can be replaced with the interface `BreadcrumbsProvider`, which is not deprecated. See also the [source of `BreadcrumbsInfoProvider`](https://github.com/JetBrains/intellij-community/blob/aea10d4404ccd132a3d8698f1aaa707d6523ce8d/xml/openapi/src/com/intellij/xml/breadcrumbs/BreadcrumbsInfoProvider.java) and [of `BreadcrumbsProvider`](https://github.com/JetBrains/intellij-community/blob/aea10d4404ccd132a3d8698f1aaa707d6523ce8d/platform/editor-ui-api/src/com/intellij/ui/breadcrumbs/BreadcrumbsProvider.java).

Reply:
Done.